cloud-etl/docs/index.rst

```diff
@@ -0,0 +1,254 @@
+.. _cloud-etl:
+  
+.. toctree::
+    :maxdepth: 2
+
+Cloud ETL Demo
+==============
+
+This demo showcases a cloud ETL leveraging all fully-managed services on
```

Comment:
maybe consider "a cloud ETL solution"... or maybe people use ETL as a noun but it reads slightly off to me.

Reply:
👍 @rspurgeon agree, and fixed